src/pybind/mgr/dashboard/services/cephfs.py

```diff
@@ -49,13 +49,14 @@ def get_dir_list(self, dirpath, level):
             dent = self.cfs.readdir(d)
             paths = [dirpath]
             while dent:
-                logger.debug("[CephFS] found entry=%s", dent.d_name)
-                if dent.d_name in ['.', '..']:
+                d_name = dent.d_name.decode('utf-8')
```

Comment:
paths should never be converted to strings unless presented to the user on a GUI/error message/output

Reply:
Done